src/Components/ProfileListPage.js

```diff
@@ -149,7 +156,16 @@ export default function ProfileListPage() {
           </Tooltip>
         )}
       </Box>
-
+      {listId.toLowerCase() !== "likes" &&
```

Comment:
Maybe we could add a variable above called `canEditBio`, similar to `canDelete`?  Then we wouldn't have to duplicate the logic of what it means for a list to be "likes" or "dislikes".